### PR TITLE
LPS-204668 - Remove unneeded h6

### PR DESCRIPTION
**Pull request by marcoscv-work**
Removing unnecessary H6 that are causing validation issues

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by ethib137** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx [hunk 1]
```diff
@@ -23,9 +23,9 @@ const KeyInputs = ({id}) => {
 				/>
 			</div>
 
-			<h6 className="font-weight-normal mb-3 mx-3">
+			<div className="h6 font-weight-normal mb-3 mx-3">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx:26]: h6 needs to come after font-weight-normal.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/common/components/fragments/Forms/Label.js [hunk 2]
```diff
@@ -9,15 +9,15 @@ import React from 'react';
 export function Label({children, className, label, name, required = false}) {
 	return (
 		<label className="align-items-center" htmlFor={name}>
-			<h6
+			<div
 				className={classNames(
-					'd-inline-block font-weight-bolder',
+					'd-inline-block h6 font-weight-bolder',
```
Comment (ethib137) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/common/components/fragments/Forms/Label.js:14]: h6 needs to come after the font-weight-bolder class.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx [hunk 3]
```diff
@@ -38,11 +38,11 @@ const KeyInputs = ({id}) => {
 					validations={[(value) => isValidIp(value)]}
 				/>
 
-				<h6 className="font-weight-normal mb-3 mx-3">
+				<div className="h6 font-weight-normal mb-3 mx-3">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx:41]: h6 needs to come after font-weight-normal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx [hunk 4]
```diff
@@ -55,11 +55,11 @@ const KeyInputs = ({id}) => {
 						validations={[(value) => isValidMac(value)]}
 					/>
 
-					<h6 className="font-weight-normal mb-3 mx-3">
+					<div className="h6 font-weight-normal mb-3 mx-3">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeyInputs/index.jsx:58]: h6 needs to come after font-weight-normal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeySelect/index.jsx [hunk 5]
```diff
@@ -37,12 +37,12 @@ const KeySelect = ({
 					required
 				/>
 
-				<h6 className="font-weight-normal ml-3 mt-1">
+				<div className="h6 font-weight-normal ml-3 mt-1">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/KeySelect/index.jsx:40]: h6 needs to come after font-weight-normal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx [hunk 6]
```diff
@@ -415,11 +415,11 @@ const RequiredInformation = ({
 										/>
 									</div>
 
-									<h6 className="font-weight-normal ml-3 mt-1">
+									<div className="h6 font-weight-normal ml-3 mt-1">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx:418]: h6 needs to come after font-weight-normal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx [hunk 7]
```diff
@@ -438,11 +438,11 @@ const RequiredInformation = ({
 									</div>
 
 									{!hasKeyComplimentary && (
-										<h6 className="font-weight-normal ml-3 mr-0 mt-1">
+										<div className="h6 font-weight-normal ml-3 mr-0 mt-1">
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx:441]: h6 needs to come after font-weight-normal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/ManageProductUsers/components/ManageUsersButton/ManageUsersButton.jsx [hunk 8]
```diff
@@ -18,7 +18,7 @@ const ManageUsersButton = ({href, title}) => (
 			title="link-only-accessible-to-current-product-users-permissions-and-roles-are-managed-separately-within-each-product"
 		/>
 
-		<h6 className="font-weight-semi-bold m-0 pr-1">{title}</h6>
+		<div className="h6 font-weight-semi-bold m-0 pr-1">{title}</div>
```
Comment (ethib137) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/ManageProductUsers/components/ManageUsersButton/ManageUsersButton.jsx:21]: h6 needs to come after font-weight-semi-bold

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/more-resources-cards/ddm-template.ftl [hunk 9]
```diff
@@ -16,9 +16,9 @@
 						</svg>
 					</#if>
 
-					<h6 class="${navItemIconId?has_content?then("pt-3", "")} title">
+					<div class="${navItemIconId?has_content?then("pt-3", "")} h6 title">
```
Comment (ethib137) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/more-resources-cards/ddm-template.ftl:19]: h6 should come first here.

**Review by ethib137 — CHANGES_REQUESTED** [review]
Thanks @marcoscv-work ... just a couple instances of sorting needed. :)

**Comment by marcoscv-work** [comment]
> Thanks @marcoscv-work ... just a couple instances of sorting needed. :)

Thanks @ethib137 ! better eye than the SF !

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by ethib137** [comment]
Hey @marcoscv-work ... I see that you force pushed changes, but when I check the changed files it still shows the sorting issues. Can you take a look?

**Comment by marcoscv-work** [comment]
rere-forced, you should see everything fine now @ethib137

**Comment by ethib137** [comment]
ci:test:relevant

**Comment by ethib137** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
rebased!

ci:test:relevant

**Comment by marcoscv-work** [comment]
ci:test:relevant

Rebased!

**Comment by ethib137** [comment]
Hey @marcoscv-work we now need to check failing tests ourselves. So, would you be able to look through [this document](https://liferay.atlassian.net/wiki/spaces/QA/pages/2222817333/How+QA+can+analyze+team+PRs) and use it to evaluate the failing tests for this PR.

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by ethib137** [comment]
Hey @marcoscv-work could you rebase this and force push... then I'll do some more work to try and verify the ci results.

**Comment by marcoscv-work** [comment]
@ethib137 sorry for the delay, the auto-rebase was a disaster.
Thanks for taking a look at this!

**Comment by ethib137** [comment]
resent here: https://github.com/liferay/liferay-portal/pull/5840

**Pull request closed without merging**